encoders/nlp/TextPaddlehubEncoder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM paddlepaddle/paddle
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+# RUN pip install -r requirements.txt
+RUN pip install jina
```

Comment:
`jina[devel]`?

Reply:
switched back to the jina base image. The `paddlepaddle/paddle` is more than 10G!